### PR TITLE
Refactor ziiaol

### PR DESCRIPTION
**Pull request by JeodC**
- Use gmloadernext.armhf
- Update patchscript

Tested on rocknix

### CODE REVIEW

File: ports/ziiaol/Zelda II Enhanced.sh [hunk 1]
```diff
@@ -21,15 +21,11 @@ get_controls
 GAMEDIR="/$directory/ports/ziiaol"
 
 # Exports
-export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs32:$GAMEDIR/libs":$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH="/usr/lib:$GAMEDIR/lib:$GAMEDIR/libs:$LD_LIBRARY_PATH"
```
Comment (Cebion) [inline ports/ziiaol/Zelda II Enhanced.sh:24]: Isn't this game armhf? /usr/lib points to aarch64 libraries.

Comment (JeodC) [reply ports/ziiaol/Zelda II Enhanced.sh:24]: For the patcher. It still boots and runs.

Comment (cdeletre) [reply ports/ziiaol/Zelda II Enhanced.sh:24]: Does the Patcher need this `/usr/lib` to be able to play the background music or something like this ? Or it totally fails without ?

Comment (JeodC) [reply ports/ziiaol/Zelda II Enhanced.sh]: I have verified this is not needed if $LD_LIBRARY_PATH is present.

**Review by kloptops — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**